package.json

```diff
@@ -19,16 +19,16 @@
   },
   "homepage": "https://github.com/voyagegroup/eslint-config-fluct",
   "peerDependencies": {
-    "eslint": "^4.0.0"
+    "eslint": "4.0.0"
```

Comment:
Why lock this field?

[`peerDependencies`](https://docs.npmjs.com/files/package.json#peerdependencies) specify the version that user project can use without some fears. It causes the problem that users cannot choose their `eslint` dependency to their preferred version.

We should not lock this field.